README.md

```diff
@@ -49,6 +49,25 @@ Given the locking on the **model** there will be some parallelization, but also
 
 Because **Zync** will keep a **log** of **events** it will be able to replay changes and recover the last state just taking last revisions of each **model** and even remove the ones that have been created before but have been deleted.
 
+## Running locally
+
+### Run a PostgreSQL server
+
+It is easy with Docker or Podman
+```
+sudo dnf install postgresql pspg
+docker run -d -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_DB=zync --name postgres10-zync docker.io/circleci/postgres:10.5-alpine
```

Comment:
OK this is adding docker as a dependency for the project... Install docker section?

But to make it the OS way you can use modules:

```suggestion
sudo dnf module enable postgresql:10
sudo dnf module switch-to postgresql:10
```

Reply:
In porte we have MySQL instructions for container as well local installation. I prefer and have tested container installation of postfres, that's why I add only it. If you have tested OS installation, I can add it as well. Let me know.

Reply:
Container is always an option but as you are using Fedora 34 in this section I'd continue with pure OS installation. Just to be consistent.

Reply:
By the way in Fedora it is podman not docker thus the section about installation of Docker :D 
I guess it is just preferable to use the system OS

Reply:
I'm not trying to write complete documentation but just add something that I am using and see working. 

btw documentation is not Fedora or Linux specific at all. It would need more work to make sections in readme and make it OS specific. For me `docker` example is a good improvement and should work across different OSes (including Fedora and `docker` alias for podman).

Adding Fedora and other OS specific information can be a future task IMO. Can we merge as is and leave further improvements for the future?

Reply:
Well there is https://github.com/3scale/zync/blob/master/INSTALL.md

You should put it there by the way instead of in the README

And in this file we already have the installation procedure for Mac OS X with a Brewfile
So I assumed this is for Fedora

Reply:
Another approach is just assume the user knows how to install different dependencies for its OS and knows how to run rails we can just list them:

- postgresql
- ruby 2.7

And it should be enough